packages/contracts-bedrock/contracts/deployment/MigrationSystemDictator.sol

```diff
@@ -123,11 +123,11 @@ contract MigrationSystemDictator is BaseSystemDictator {
             abi.encodeCall(OptimismPortal.initialize, ())
         );
 
-        // Upgrade and initialize the L1CrossDomainMessenger.
-        config.globalConfig.proxyAdmin.upgradeAndCall(
+        // Upgrade the L1CrossDomainMessenger. No initializer because this is
```

Comment:
cc @tynes and @maurelian I pinged you about this already but I'd like a double-check here.